maintainers/README.md

```diff
@@ -69,52 +69,73 @@ checks should be performed:
 - If the user has specified a GPG key, verify that the commit is
   signed by their key.
 
-  First, validate that the commit adding the maintainer is signed by
-  the key the maintainer listed. Check out the pull request and
-  compare its signing key with the listed key in the commit.
-
-  If the commit is not signed or it is signed by a different user, ask
-  them to either recommit using that key or to remove their key
-  information.
-
-  Given a maintainer entry like this:
-
-  ``` nix
-  {
-    example = {
-      email = "user@example.com";
-      name = "Example User";
+  For example, with an entry as follows:
+
+  ```nix
+    SomeMaintainer = {
+      name = "Some Maintainer";
+      email = "some.maintainer@example.com";
+      github = "some-maintainer";
+      githubId = 1234567890;
       keys = [{
-        fingerprint = "0000 0000 2A70 6423 0AED  3C11 F04F 7A19 AAA6 3AFE";
+        fingerprint = "21E5 9F74 B993 9CAE 09E1  156A C894 8ADE 4250 5525";
```

Comment:
Would be best to offer a suggestion that complies with `nixfmt`.

For a single item would be:
> keys = [ { fingerprint = "E10B D192 9231 08C7 3C35 7EC3 83E0 3DC6 F383 4086"; } ];